### PR TITLE
CommonServerPython - IAM Classes

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28270

## Description
Adding the class IAMCommandHelper used in integrations that are part of the IAM 

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by anara123 — CHANGES_REQUESTED** [review]
Add unit tests

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -4570,3 +4586,228 @@ def to_display(self):
...
+
+
+class IAMVendorActionResult:
+    """ This class is used in IAMUserProfile class to represent actions data.
+    :return: None
+    :rtype: ``None``
+    """
+
+    def __init__(self, success=None, active=None, iden=None, username=None, email=None, error_code=None,
+                 error_message=None, details=None, skip=False, skip_reason=None, action=None):
```
Comment (anara123) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4621]:
```suggestion
    def __init__(self, success, active, email, action, iden=None, username=None, error_code=None,
                 error_message=None, details=None, skip=False, skip_reason=None):
```

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -4570,3 +4586,228 @@ def to_display(self):
...
+            'Contents': outputs,
+            'HumanReadable': readable_output,
+            'EntryContext': entry_context
+        }
+
+        return return_entry
+
+    def set_result(self, success=None, active=None, iden=None, username=None, email=None, error_code=None,
+                   error_message=None, details=None, skip=False, skip_reason=None, action=None):
+        """ Sets the outputs and readable outputs attributes according to the given arguments.
```
Comment (anara123) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4755]:
```suggestion
    def set_result(self, success, active, email, action, iden=None, username=None, error_code=None,
                   error_message=None, details=None, skip=False, skip_reason=None):
        """ Sets the outputs and readable outputs attributes according to the given arguments.
```

**Review by anara123 — CHANGES_REQUESTED** [review]

**Review by anara123 — APPROVED** [review]

**Comment by kirbles19** [comment]
@ohaim1008 @dantavori Done.

**Pull request merged**